src/client.js

```diff
@@ -118,6 +119,15 @@ class Client {
   }
 
   /**
+   * A wrapper around the constructor, for backwards compatability.
```

Comment:
yep. that clears things up. why not just use `.buildClient` in our examples then?